src/components/textarea/TextArea.tsx

```diff
@@ -80,10 +82,20 @@ export class TextArea extends React.Component<ITextAreaProps, {}> {
                 disabled={this.props.disabled}
                 className={this.props.className}
                 value={this.props.value}
-                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => this.props.onChange(e)}
+                onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => this.handleOnChange(e)}
             />
         );
     }
+
+    private handleOnChange(e: React.ChangeEvent<HTMLTextAreaElement>) {
+        if (this.props.onChange) {
+            this.props.onChange(e);
+        }

```

Comment:
`callIfDefined(this.props.onChange, e);`

Reply:
oh nice!